packages/terra-application-links/src/tabs/_Tab.jsx

```diff
@@ -17,48 +30,113 @@ const propTypes = {
    */
   text: PropTypes.string.isRequired,
   /**
-   * The click callback of the tab.
+   * The click callback of the tab..
```

Comment:
Is the extra period for.... Drama?

![](https://media.giphy.com/media/HyteoA8MexIac/giphy.gif)

Reply:
I'll add one more.